### DISCUSSION TITLE
lsp is not working

### DISCUSSION THREAD

**Discussion by Mukku-1** [helix-editor/helix#7722] [Troubleshooting]
cannot find binary path  
how i can add binary path 
i am using debian 12 
i install it from flatpak

**Comment by Mukku-1** [comment]
![Screenshot from 2023-07-24 11-16-05](https://github.com/helix-editor/helix/assets/112679989/a8ca1ad6-7660-465a-bd2a-17380f97a8c6)

**Reply by arifardia** [reply]
have you installed typescript-language-server?

**Reply by Mukku-1** [reply]
no how i can install

**Reply by Mukku-1** [reply]
helix come with prebuilt

**Reply by Mukku-1** [reply]
i thought can tell me how i can install it

**Reply by arifardia** [reply]
just run that command and lsp will work
`npm install -g typescript typescript-language-server`

[you can read this to install lsp](https://github.com/helix-editor/helix/wiki/How-to-install-the-default-language-servers)

**Reply by fizzyhuts** [reply]
Even after running that command, the flatpak still cannot recognize where the lsp is, because the flatpak cannot access the home folder. So how to make flatpak'd Helix recognize lsp's?
